yarn-project/boxes/private-token/src/contracts/src/main.nr

```diff
@@ -95,7 +95,7 @@ contract PrivateToken {
     // Note 1: Needs to be defined by every contract producing logs.
     // Note 2: Having it in all the contracts gives us the ability to compute the note hash and nullifier differently for different kind of notes.
     unconstrained fn compute_note_hash_and_nullifier(contract_address: Field, nonce: Field, storage_slot: Field, preimage: [Field; VALUE_NOTE_LEN]) -> [Field; 4] {
-        let note_header = NoteHeader { contract_address, nonce, storage_slot };
+        let note_header = NoteHeader::new(contract_address, nonce, storage_slot);
```

Comment:
Changing the initialisation to use a method so that we don't have to add `is_transient` now and have to remove it later.